packages/table/src/table.vue

```diff
@@ -306,7 +306,9 @@
 
       tooltipEffect: String,
 
-      spanMethod: Function
+      spanMethod: Function,
+
+      selectOnIndeterminate: Boolean
```

Comment:
Hi, sorry for the delay. I think this prop should default to `true` so that the default behavior is not changed. Otherwise it's kind of a breaking change.

Reply:
No problem, thanks for the review.

It would be a breaking change, yes, but arguably a better default behavior? Regardless, I've updated it to default to `true`.